src/components/HelpCenterBar/HelpCenterBar.test.tsx

```diff
@@ -132,7 +132,7 @@ describe('<HelpCenterBar/>', () => {
     const dialog = screen.getByRole('dialog', { name: 'About NeoBoard' });
 
     await userEvent.click(
-      within(dialog).getByRole('button', { name: 'Close' })
+      within(dialog).getAllByRole('button', { name: 'Close' })[0]
```

Comment:
It's unfortunate that we have two buttons with the same name, but I guess that's how it is...

Reply:
No. If the design asks for that, it's fine.